### PR TITLE
Move build page into pipeline route

### PR DESCRIPTION
**Pull request by petey**
Requires/includes: https://github.com/screwdriver-cd/ui/pull/27

Migrates the build page into the pipeline route `/pipelines/:pid/build/:bid`
The old build page will redirect to the new one.
Added new fancy header.

screen shot 2016-09-16 at 7 44 02 pm

### CODE REVIEW

**Comment by petey** [comment]
Added new home page.

screencapture-localhost-4200-1474231282098

File: app/components/build-nav/styles.scss [hunk 1]
```diff
@@ -4,14 +4,19 @@
...
+& {
+  background: url('/assets/main_bkgrd_home.png') transparent;
+  background-size: cover;
+  background-position: 0 -130px;
+}
 
 .pure-menu-horizontal {
   padding: 0 25px;
-  @include backgroundGradient($ycolor-blue-10b, #666);
+  // @include backgroundGradient($ycolor-blue-10b, #666);
```
Comment (stjohnjohnson) [inline app/components/build-nav/styles.scss]: Should probably clean up these comments, but we can do so in another PR
